Assets/Mirage/Runtime/NetworkManager.cs

```diff
@@ -27,5 +28,22 @@ public class NetworkManager : MonoBehaviour
         /// <para>This is set True in StartServer / StartClient, and set False in StopServer / StopClient</para>
         /// </summary>
         public bool IsNetworkActive => Server.Active || Client.Active;
+
+        /// <summary>
+        /// helper enum to know if we started the networkmanager as server/client/host.
+        /// </summary>
+        public NetworkManagerMode mode
+        { get
+            {
+                if (!Server.Active && !Client.Active)
+                    return NetworkManagerMode.Offline;
+                else if (Server.Active && Client.Active)
+                    return NetworkManagerMode.Host;
+                else if (Server.Active)
+                    return NetworkManagerMode.ServerOnly;
```

Comment:
maybe we should use `Server` instead of `ServerOnly`

Reply:
changing it would need to be written in the migration guide which isnt a huge deal.